sources/packages/web/src/views/student/AppStudent.vue

```diff
@@ -173,14 +199,28 @@ export default defineComponent({
     });
 
     onMounted(async () => {
-      const { isFulltimeAllowed } = await AppConfigService.shared.config();
-      if (!isFulltimeAllowed) {
-        menuItems.value.forEach((item, index) => {
-          if (item.title === "Overawards Balance") {
-            menuItems.value.splice(index, 1);
-          }
-        });
+      const mediaQuery = window.matchMedia("(max-width: 1060px)");
```

Comment:
Can we use the `smAndDown` not a any particular width based on current testing unless there is a blocker. 

![image](https://github.com/user-attachments/assets/fc1615aa-01db-46dc-b769-74e5b9b68199)